frontend/src/ui/ErrorBoundary.tsx

```diff
@@ -0,0 +1,47 @@
+import { exception } from 'console';
+import React from 'react';
+
+interface errorBoundaryState {
+  hasError: boolean;
+}
+
+export const ThrowsError = () => {
+  throw exception('This is failing.');
+};
+
+export default class ErrorBoundary extends React.Component<
```

Comment:
I think this is part of another PR. Can we remove this?

Reply:
removed as part of updates.